packages/website/docs/SingleSelect.md

```diff
@@ -1,24 +1,49 @@
-## Guidelines
+# Best Practices
 
-### General
+Use single-select to allow user to select one option from a **long list that can be filtered**. Single-selects are more appropriate when the available list of options is very long or when the space is limited.
 
--   If space allows, and there are no more than 7 options to choose from, consider using [radio buttons](#/components/Radio) instead.
--   Keep option labels as short as possible in order to limit width to a minimum.
+Use alphanumerical order unless a more suited ordering rational applies (such as sizes or security levels).
 
-### Options Sorting
+Always include the ability to filter the list when there are over 20 options.
 
--   Always put the most common items at the top of the list. If no option is more common than the others, use alphanumeric or chronological order.
+## Labeling
 
-### Label
+Keep titles and labels short, **under one line**.
 
--   In most cases, you should include a label to identify the component.
+### Titles
+
+The title is used to indicate the type of information to be provided.
+
+Use descriptive labels without action verbs.
+
+-   E.g. Use "Favorite drink" instead of "Select your favorite drink".
+
+### Labels
+
+Labels are used to name each option. The width of the input should allow to fully display the name of the chosen option.
+
+Use a consistent writing style for all options in the list.
+
+## Help text and instructions
+
+The placeholder text should indicate the type of information that must be selected with an action verb.
+
+-   E.g. Use "Select a condiment" instead of "Select an option".
+
+## Feedback and validation
+
+Allow the addition of custom values only when it doesn't increase the risk of failure and when the availability of all possible options in the list can not be ensured.
```

Comment:
I'd like that the second part with "when there may be options other than those listed" is very clear about "we can't realisticaly list all possible options because there are too many possibilities" 

"may be other options" feels liek we could just show you 5 random choices...